src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/ProjectEngineFactory_UnsupportedTest.cs

```diff
@@ -5,13 +5,19 @@
 
 using System;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Test.Common;
 using Xunit;
+using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
 {
-    public class ProjectEngineFactory_UnsupportedTest
+    public class ProjectEngineFactory_UnsupportedTest : TestBase
     {
+        public ProjectEngineFactory_UnsupportedTest(ITestOutputHelper testOutput) : base(testOutput)
```

Comment:
This was just missed. I'm also in the "`: base`" on the separate line crew. It's so sad that this was so close to the top of the commit because I was super-careful to catch everything, but my eyes scanned past this one. :smile:

Reply:
I went ahead and searched for all cases with `: base` appears on the same line as a constructor parameter list in the test folder and updated them. (Regex used: `\)\s+\:\s+base\(`)